plugins/builtin/target/gce-mig/plugin/gce.go

```diff
@@ -0,0 +1,157 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/nomad-autoscaler/sdk"
+	"github.com/hashicorp/nomad-autoscaler/sdk/helper/scaleutils"
+	"github.com/mitchellh/go-homedir"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/option"
+	"io/ioutil"
+	"os"
+	"time"
```

Comment:
nit: could the imports be separated; stdlib top and other below, with the two sections separated by a newline and each section sorted alphabetically?